.moai/specs/SPEC-REF-SEO-ABSORB-001/acceptance.md

```
@@ -0,0 +1,721 @@
+grep -n 'tier: optional-pack:frontend' internal/template/catalog.yaml | grep -c .
+grep -n -A1 'name: moai-ref-seo' internal/template/catalog.yaml
+go test ./internal/template/ -run 'TestCatalogTierValid|TestWorkflowTriggerCoverage' -count=1 -v 2>&1 \
+  | grep -E '^(--- PASS|--- FAIL|ok|FAIL)'
+grep -rn 'required-skills' .claude/skills/moai/workflows/ internal/template/templates/.claude/skills/moai/workflows/ | wc -l
+```
+
+기대: 신규 엔트리의 `tier`가 `optional-pack:frontend`, `--- PASS: TestCatalogTierValid` 및 `--- PASS: TestWorkflowTriggerCoverage` 두 행이 출력에 등장, `required-skills` 매칭 수 `0`.
+
+근거 baseline (plan-phase 실측):
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)